intellij/ast/src/test/kotlin/motif/ast/intellij/IntelliJAnnotationTest.kt

```diff
@@ -15,28 +15,24 @@
  */
 package motif.ast.intellij
 
-import com.intellij.pom.java.LanguageLevel
+import com.intellij.openapi.projectRoots.impl.JavaAwareProjectJdkTableImpl
 import com.intellij.psi.PsiElementFactory
-import com.intellij.testFramework.LightProjectDescriptor
-import com.intellij.testFramework.fixtures.LightCodeInsightFixtureTestCase
-import motif.intellij.testing.InternalJdk
+import com.intellij.testFramework.fixtures.DefaultLightProjectDescriptor
+import com.intellij.testFramework.fixtures.LightJavaCodeInsightFixtureTestCase
 import org.assertj.core.api.Assertions.assertThat
 import org.intellij.lang.annotations.Language
 
-class IntelliJAnnotationTest : LightCodeInsightFixtureTestCase() {
+class IntelliJAnnotationTest : LightJavaCodeInsightFixtureTestCase() {
 
   lateinit var psiElementFactory: PsiElementFactory
 
   override fun setUp() {
     super.setUp()
-
     psiElementFactory = PsiElementFactory.SERVICE.getInstance(project)
   }
 
-  override fun getProjectDescriptor(): LightProjectDescriptor {
-    return object : ProjectDescriptor(LanguageLevel.HIGHEST) {
-      override fun getSdk() = InternalJdk.instance
-    }
+  override fun getProjectDescriptor() = DefaultLightProjectDescriptor {
+    JavaAwareProjectJdkTableImpl.getInstanceEx().internalJdk
```

Comment:
`internalJdk` is deprecated and longer term, I think we need to start using the "fake jdk" provided by the test case instead of being reliant on the environment

Reply:
Are we not able to use `LightJavaCodeInsightFixtureTestCase.JAVA_1X` instead?

Reply:
We are able to use those (and that was my initial approach), but the problem that I ran into was that some tests fail with strange errors such as:
* expecting `java.lang.String`, got `String`
* expecting superclass `java.lang.Object`, got null
(presumably due to the jdk is a fake jdk? or maybe it was something else)

I do agree we should probably use `LightJavaCodeInsightFixtureTestCase.JAVA_XX` in the long run, but we may need to examine those test failures more closely to see if we can allow those discrepancies or will we weaken our tests by doing so, and I wasn't ready to do that in this pull request.